demos/ota/ota_demo_core_http/ota_demo_core_http.c

```diff
@@ -1625,14 +1594,15 @@ static void prvDisconnectFromMQTTBroker( void )
     /* End TLS session, then close TCP connection. */
     ( void ) SecureSocketsTransport_Disconnect( &networkContextMqtt );
 }
+/*-----------------------------------------------------------*/
 
-static int32_t connectToS3Server( NetworkContext_t * pNetworkContext,
-                                  const char * pUrl )
+static int32_t connectToS3Server( NetworkContext_t * pxNetworkContext,
```

Comment:
Should the return type of this be BaseType_t to match the returnStatus variable that you return?

Reply:
`EXIT_STATUS` is generally denoted by int, needn't be long.